code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -427,6 +429,29 @@ GLOBAL_LIST_INIT(cardboard_recipes, list (
 	recipes = GLOB.cardboard_recipes
 	return ..()
 
+
+/*
+ * soil
+ */
+GLOBAL_LIST_INIT(soil_recipes, list (
+	new /datum/stack_recipe("pile of dirt", /obj/machinery/hydroponics/soil, 3, time = 1 SECONDS, one_per_turf = TRUE, on_floor = TRUE)
```

Comment:
Is there going to be more recipes that uses soil in the future? If not this could be acquired by clicking on the soil instead of a recipes menu.

Reply:
I'm not sure, but I wanted to keep it open. It seems a waste to restrict it since it isn't exactly a specialized material. Maybe a wall recipe for some minecraft RP.